android/app/src/main/java/com/mattermost/rnbeta/ReceiptDelivery.java

```diff
@@ -64,21 +65,24 @@ protected static void execute(String serverUrl, String token, String ackId, Stri
             return;
         }
 
-        final OkHttpClient client = new OkHttpClient();
-        final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
-        RequestBody body = RequestBody.create(JSON, json.toString());
-        Request request = new Request.Builder()
-                .header("Authorization", String.format("Bearer %s", token))
-                .header("Content-Type", "application/json")
-                .url(String.format("%s/api/v4/notifications/ack", serverUrl.replaceAll("/$", "")))
-                .post(body)
-                .build();
-
-        try {
-            client.newCall(request).execute();
-        } catch (Exception e) {
-            Log.e("ReactNative", "Receipt delivery failed to send");
+        final HttpUrl url = HttpUrl.parse(
```

Comment:
I thought we were going to detect and parse the old format and fallback to the new one

Reply:
Yeah this avoids the crash but we don’t get the ack back